### PR TITLE
Fixes #11754: Update prerequisite and document windows plugin installation on server and agent

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/11754

### CODE REVIEW

File: 95_windows/10_server_installation.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[[dsc-server-installation]]
+=== Install Windows DSC plugin on the server
+
+The installation and upgrade processes are exactly the same.
+Download the *rpkg* file, and run, on the server:
+
+====
+
+rudder-pkg install-file rudder-plugin-dsc-<Rudder version>-<plugin version>.rpkg
```
Comment (ncharles) [inline 95_windows/10_server_installation.txt]: can you put the full path to the command ?

File: 95_windows/20_agent_installation.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+====
+C:\Program Files\Rudder\etc\policy-server.conf
+====
+
+Before running the installer with:
+
+====
+
+rudder-agent-dsc-<Rudder version>-<plugin version>.exe \S
```
Comment (ncharles) [inline 95_windows/20_agent_installation.txt]: Thank you <3

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Pull request merged**